packages/eslint-plugin-template/src/utils/get-attribute-value.ts

```diff
@@ -13,35 +15,36 @@ import { PROPERTY } from './constants';
 // And this is not:
 // <input [disabled]="disabled">
 //        ~~~~~~~~~~~~~~~~~~~~~
-export function getAttributeValue<T extends string | number | boolean = string>(
-  node: TmplAstElement,
+export function getAttributeValue(
+  { attributes, inputs }: TmplAstElement,
   attributeName: string,
-): T | string | null | typeof PROPERTY {
-  const attribute = node.attributes.find(
-    (attribute: TmplAstTextAttribute) => attribute.name === attributeName,
+): unknown {
```

Comment:
I used `unknown` here because the generic can lead us to a mistake as you can't really know what type is present in `value`. The best way to check this is checking the result via `typeof` or `instanceof`.